EOS_DISABLE Scripting Define not compiling

**Describe the bug**
I'm currently trying to do a build with the EOS plugin disabled. I followed the [instructions for disabling the plugin](https://github.com/PlayEveryWare/eos_plugin_for_unity/blob/stable/docs/disable_plugin_per_platform.md), but get compilation errors once hitting "Apply" in the editor.

**To Reproduce**
1. Install v3.2.0 from tarball
2. Add "EOS_DISABLE" to Scripting Define Symbols

**Expected behavior**
EOS plugin is completely disabled and compiles correctly.

**Desktop**
 - OS: Windows x64
 - Unity Version - 2022.3.16f1
 - Plugin Version - v3.2.0

**Relevant errors**
`
Library\PackageCache\com.playeveryware.eos@4e0c43fa3abb\Runtime\Core\EOSManager.cs(79,32): error CS0246: The type or namespace name 'Epic' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.playeveryware.eos@4e0c43fa3abb\Runtime\Core\EOSManager.cs(80,27): error CS0246: The type or namespace name 'Epic' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.playeveryware.eos@4e0c43fa3abb\Runtime\Core\EOSManager.cs(81,30): error CS0246: The type or namespace name 'Epic' could not be found (are you missing a using directive or an assembly reference?)
`

--- a/Assets/Plugins/Source/Core/EOSManager.cs
+++ b/Assets/Plugins/Source/Core/EOSManager.cs
@@ -73,12 +73,13 @@ namespace PlayEveryWare.EpicOnlineServices
     using LoginCallbackInfo = Epic.OnlineServices.Auth.LoginCallbackInfo;
     using LoginOptions = Epic.OnlineServices.Auth.LoginOptions;
     using LoginStatusChangedCallbackInfo = Epic.OnlineServices.Auth.LoginStatusChangedCallbackInfo;
-#endif
+
     using Utility;
     using JsonUtility = PlayEveryWare.EpicOnlineServices.Utility.JsonUtility;
     using LogoutCallbackInfo = Epic.OnlineServices.Auth.LogoutCallbackInfo;
     using LogoutOptions = Epic.OnlineServices.Auth.LogoutOptions;
     using OnLogoutCallback = Epic.OnlineServices.Auth.OnLogoutCallback;
+#endif
 
     /// <summary>
     /// One of the responsibilities of this class is to manage the lifetime of

